__tests__/pages/_app.test.js

```diff
@@ -0,0 +1,14 @@
+import { render } from '@testing-library/react';
```

Comment:
Because it mirrors the _app.js file and that file is named that way because next requires it to be called that. See https://nextjs.org/docs/advanced-features/custom-app

Reply:
Yeah, that's what I figured out later and forgot to remove the comment before submitting them.